sysobject.ids

```diff
@@ -7349,6 +7349,7 @@
 25053.3.1.4.100	Ruckus	NETWORKING	R610
 25053.3.1.4.103	Ruckus	NETWORKING	T310c
 25053.3.1.5.2	Ruckus	NETWORKING	ZoneDirector
+25053.3.1.5.15 Ruckus	NETWORKING R550
```

Comment:
Fields separator must be a tabulation, it is not between oid suffix and manufacturer:
```suggestion
25053.3.1.5.15	Ruckus	NETWORKING R550
```